Api/test/userTest.js

```diff
@@ -0,0 +1,61 @@
+import chai from "chai";
+import chaiHttp from "chai-http";
+import {
+  res
+} from "express";
+const expect = chai.expect;
+
+//Assertion
+chai.should();
+chai.use(chaiHttp);
+
+// To do
+describe('User API', () => {
+  //Testing Different Routes 
+  // Get route
+  describe("Get /api/src/routes/userRoute", () => {
```

Comment:
there's no route like this, `/api/src/routes/userRoute`, is there? 
```suggestion
  describe("Get /api/src/routes", () => {
```

Reply:
Done.